Readme.md

```diff
@@ -148,6 +166,41 @@ The current lead maintainer is [Douglas Christopher Wilson](https://github.com/d
 
 [List of all contributors](https://github.com/expressjs/express/graphs/contributors)
 
+### Collaborators
+
+* [Jon Church](https://github.com/jonchurch) -
+  **Jon Church** <<me@jonchurch.com>>
+* [Wes Todd](https://github.com/wesleytodd) -
+  **Wes Todd** <<wes@wesleytodd.com>>
+
+### Technical Committee
+
+* [Ulises Gascon](https://github.com/UlisesGascon) -
+  **Ulises Gascon** <<ulisesgascongonzalez@gmail.com>>
```

Comment:
```suggestion
  **Ulises Gascon** <<ulisesgascongonzalez@gmail.com>> (he/him)
```
 I will suggest to include the option to include pronouns, as [Node.js does](https://github.com/nodejs/node/blob/main/README.md?plain=1#L229)